api/swagger.yml

```diff
@@ -1718,6 +1726,8 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         404:
           $ref: "#/components/responses/NotFound"
+        409:
```

Comment:
In Git you can `git tag --force` to overwrite an existing tag - In lakectl this happened implicitly for all tag creations, but it won't anymore. Maybe add `--force` to the lakcetl command? (could simply delete and recreate it to simulate the current behavior).

(This could be done in another PR)

Reply:
Done.